File: week6-Python/string.py
from cs50 import get_string

s = get_string("Input: ")
print("Output: ", end = "")
# Original
for i in range(len(s)):
    print(s[i], end= "")
print()

# Progress1
for c in s:
    print(c, end = "")
print()

File: week2-Arrays/string.c
#include <cs50.h>
#include <stdio.h>
#include <string.h>

int main(void)
{
    string s = get_string("Input:  ");
    printf("Output:  ");
    // original
    // for (int i =0; s[i] != '\0'; i++)
    // progress 1
    // int n = strlen(s);
    // progress 2
    for (int i =0, n = strlen(s); i < n; i++)
    {
        printf("%c",s[i]);
    }
    printf("\n");
}

//  clang -o string string.c -lcs50